code/game/machinery/iv_drip.dm

```diff
@@ -13,7 +13,9 @@
 	var/obj/item/reagent_containers/beaker
 	var/static/list/drip_containers = typecacheof(list(/obj/item/reagent_containers/blood,
 									/obj/item/reagent_containers/food,
-									/obj/item/reagent_containers/glass))
+									/obj/item/food, //I will inject a slice of pizza into my arm and you cannot stop me
```

Comment:
How does injecting food work when it runs out of reagents? Does taking a bite cause it just to vanish entirely?
May need a deletion upon running out of reagents for this case.